libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/AccountsManager.java

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import androidx.annotation.NonNull;
+
+public interface AccountsManager {
```

Comment:
what's the benefit of this wrapper interface? shouldn't we be dealing with `MapboxAccounts` directly? why this interface is public?